torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -1191,15 +1191,17 @@ void initNvFuserPythonBindings(PyObject* module) {
       "permute",
       [](nvfuser::FusionDefinition::Operators& self,
          nvfuser::Tensor arg,
-         std::vector<int64_t>& permutation) -> nvfuser::Tensor {
+         std::vector<int64_t>& order) -> nvfuser::Tensor {
         nvfuser::FusionDefinition* fd = self.fusion_definition;
         nvfuser::Tensor output = fd->defineTensor();
         self.fusion_definition->defineRecord(new nvfuser::PermuteOpRecord(
             {fd->recordingState(arg())},
             {fd->recordingState(output())},
-            permutation));
+            order));
         return output;
       },
+      py::arg("arg"),
+      py::arg("order"),
```

Comment:
If we would like nvFuser's Python frontend to mimic PyTorch's API where possible when this better be named as `dims`, it's of course less descriptive than `order`.
https://pytorch.org/docs/stable/generated/torch.permute.html

Reply:
I'll change this.